package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "eslint '**/*.{js,ts}'",
     "prebuild": "npm run clean",
     "start": "ts-node src/main.ts",
-    "test": "jest --passWithNoTests"
+    "test": "glob -c 'tsx --test' 'src/**/*.test.ts'"
```

Comment:
Due to the limitations of the node test runner, it is not possible to run a recurrent `.test.ts` file without using `glob` package.